src/lineage/templates/clean_template.md

```diff
@@ -9,6 +9,39 @@ Upstream repository: [`{{ remote_url }}`]({{ remote_url }})
 Check the changes in this pull request to ensure they won't cause issues with
 your project.
 
+## ✅ Pre-approval checklist ##
+
+<!-- Remove any of the following that do not apply. -->
+<!-- Draft PRs should have one or more unchecked boxes. -->
+<!-- If you're unsure about any of these, don't hesitate to ask. -->
+<!-- We're here to help! -->
```

Comment:
Say the quiet part out loud, since these are Lineage PRs:
```suggestion
- Remove any of the following that do not apply.
- If you're unsure about any of these, don't hesitate to ask. We're here to help!
```